src/bundle/Resources/config/services.yml

```diff
@@ -127,6 +127,18 @@ services:
         tags:
             - {name: kernel.event_subscriber, priority: -250}
 
+    EzSystems\EzPlatformAdminUi\EventListener\CredentialsExpiredListener:
```

Comment:
Does "credentials" naming already exist in AdminUI or is it fresh, based on old `isCredentialsNonExpired` in Kernel?
Asking because in general this is about just password, not all credentials.

Reply:
It's fresh, based on `\Symfony\Component\Security\Core\Exception\CredentialsExpiredException`.

Reply:
> It's fresh, based on `\Symfony\Component\Security\Core\Exception\CredentialsExpiredException`.

Hmm, ok, I won't argue with that then, but only because it would be confusing to use a different one here.
:+1: